### PR TITLE
QuestionAnsweringBERTTest

### PR DESCRIPTION
**Pull request by yirawan1996**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by szha** [comment]
Job PR-409-1 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-409/1/index.html

**Comment by Innixma** [comment]
Hi @yirawan1996,

Could you please add a description to this PR and the reason why you wish to merge it into AutoGluon?

File: autogluon/task/question_answering/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print("importing...")
```
Comment (zhanghang1989) [inline autogluon/task/question_answering/__init__.py:1]: This looks like a debug code.

**Comment by Innixma** [comment]
It is still unclear to me the reason why this PR has been requested and if any steps are being made to move it forward. Can someone please clarify within this PR?

**Comment by szha** [comment]
Job PR-409-2 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-409/2/index.html

**Comment by Innixma** [comment]
It does not appear that progress is being made, so I am closing this PR. Please re-open the PR with additional information added to continue the PR process.

**Pull request closed without merging**